lerna.json

```diff
@@ -1,7 +1,4 @@
 {
-  "packages": [
-    "examples/sqrl-example-functions",
-    "packages/*"
-  ],
+  "useWorkspaces": true,
```

Comment:
this tells lerna to rely on native monorepo bootstrapping rather than `lerna bootstrap`